### PR TITLE
fix(annotation): fix regionFilter not work on sub view

### PR DESCRIPTION
**Pull request by zqlu**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: src/chart/controller/base.ts [hunk 1]
```diff
@@ -15,13 +15,21 @@ export abstract class Controller<O = unknown> {
...
   protected components: ComponentOption[] = [];
 
   constructor(view: View) {
     this.view = view;
+    if (view.parent) {
+      this.ancestorView = view.parent;
+      while (this.ancestorView.parent) {
+        this.ancestorView = this.ancestorView.parent;
+      }
+    }
```
Comment (hustcc) [inline src/chart/controller/base.ts:32]: 不用存到 this 上，直接在需要使用的地方调用：

```ts
this.view.getRootView().once('', () => {});
```

Comment (zqlu) [reply src/chart/controller/base.ts]: done

File: tests/bugs/sub-view-region-filter-spec.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  view.axis('date', {
+    grid: null,
+  });
+
+  // Step 3：创建图形语法，绘制柱状图，由 genre 和 sold 两个属性决定图形位置，genre 映射至 x 轴，sold 映射至 y 轴
+  view.area().position('date*sold').style({
+    fillStyle: 'l(90) 0:rgba(0, 85, 255, 1) 1:rgba(0, 85, 255, .13)',
+  });
+
+  // TODO: regionFilter 不生效
```
Comment (hustcc) [inline tests/bugs/sub-view-region-filter-spec.ts:57]: 这个 todo 去掉呗~

Comment (zqlu) [reply tests/bugs/sub-view-region-filter-spec.ts]: 已去掉

**Pull request merged**